product_secondary_unit/tests/test_secondary_unit_mixin.py

```diff
@@ -82,7 +82,7 @@ def test_product_secondary_unit_mixin(self):
         fake_model.write(
             {"secondary_uom_qty": 1, "secondary_uom_id": self.secondary_unit_box_10.id}
         )
-        fake_model.flush()
+        fake_model.env.cr.flush()
```

Comment:
@alexey-pelykh Is this still necessary?

Reply:
@rousseldenis I think I have a memory of it being still needed. Any suggestions otherwise?

Reply:
Removing it :sweat_smile:

Reply:
Well, let's see what happens :)

Reply:
@rousseldenis see, tests are failing without

Reply:
👍